rclc/src/rclc/executor.c

```diff
@@ -222,6 +222,7 @@ rclc_executor_add_subscription(
   executor->handles[executor->index].subscription = subscription;
   executor->handles[executor->index].data = msg;
   executor->handles[executor->index].callback = callback;
+  executor->handles[executor->index].callback_type = CB_WITHOUT_REQUEST_ID;
```

Comment:
Why do you want to change this? Default is `CB_UNDEFINED` 

The type `CB_WITH_REQUEST_ID`  and `CB_WITHOUT_REQUEST_ID`is currently used for services/clients.

Reply:
I would also prefer  `CB_WITHOUT_CONTEXT`

Reply:
The subscriptions without context need *something* in their `callback_type` that is well-defined, but has neither request-id nor context.   `CB_WITHOUT_CONTEXT` sounds more generic, but is no more informative than `CB_WITHOUT_REQUEST_ID`, and would require another enum value.
(excuse me if this argument is already settled elsewhere, I'm catching up on the recent activity)

Reply:
Sure, but `CB_WITHOUT_REQUEST_ID` only makes sense for services and clients. `CB_WITHOUT_CONTEXT` is informative, that it is a normal callback. If you prefer another wording, maybe `CB_DEFAULT` might be an alternative.